regression_and_census.R

```diff
@@ -9,9 +9,166 @@
 # old code-  I M P O R T I N G   C E N S U S   D A T A ---------------------------------------
 
 library(latticeExtra)
-install.packages("gee")
+library(gee)
+library(geepack)
+library(MASS)
+library(broom)
 # tidyverse
 library(dplyr)
+library(readr)
+library(tidyr)
+library(magrittr)
+library(stringr)
+library(purrr)
+# plotting libraries
+library(ggplot2)
+library(ggmap)
+library(gridExtra)
+
+
+
+# Load data ---------------------------------------------------------------
+
+clean_weekly_data_dir <- "." #"/Users/funmiarogbokun/Desktop/CZ Biohub Contractor/Wildfires & Covid Project/Manuscript/Clean_final_datasets/Weekly_data"
```

Comment:
It's best not to change working directories in a R script just to read in data.
Here, I've separated out the directory where a data file lives, so that someone else can run it easily by changing this directory.

See below for the use of `file.path` to join the directory name with the data file name.